RallyHereIntegration/Source/RallyHereIntegration/Private/RH_GameInstanceSubsystem.cpp

```diff
@@ -80,6 +90,23 @@ void URH_GameInstanceSubsystem::Deinitialize()
 	ConfigSubsystem = nullptr;
 	SessionSearchCache = nullptr;
 	MatchmakingCache = nullptr;
+
```

Comment:
Interesting, I was kinda afraid of this, while my branch didn't show this as a change as it is part of pull #3, its still a change here currently, not sure how to handle this in future, I guess I need to do better stashing

Reply:
Would be nice if we could have these changes removed from this CL, but I am not too worried if they make it in with it.